### PR TITLE
Modify S1075: Fix resources link format

### PR DESCRIPTION
**Pull request by leonardo-pilastri-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Comment by leonardo-pilastri-sonarsource** [comment]
The website that the URL points to is still down, so I cannot evaluate at this time which languages would actually benefit from including the resource in their rule page.

File: rules/S1075/resources.adoc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 == Resources
 
-* Carnegie Mellon University - https://wiki.sei.cmu.edu/confluence/x/OjdGBQ[CERT, MSC03-J. Never hard code sensitive information]
+* CERT - https://wiki.sei.cmu.edu/confluence/x/OjdGBQ[Never hard code sensitive information]
```
Comment (nicolas-gauthier-sonarsource) [inline rules/S1075/resources.adoc:3]: Hi! I'm so sorry for the incorrect link I gave you!
I found a new working link for Java: 
https://wiki.sei.cmu.edu/confluence/display/java/MSC03-J.+Never+hard+code+sensitive+information
It seems Java-specific to me, and it shows a security concern we don't have in the rule description as we focus on testing difficulties.
The metadata of the rule has a reference to this MSC03-J:
```
"securityStandards": {
    "CERT": [
      "MSC03-J."
    ]
  }
```
IMO, you could remove all references to that or add some information in the description to justify the resource, like: `In addition, hard-coded URI can be considered sensitive information. It should not be stored in the code`.
WDYT?

**Review by nicolas-gauthier-sonarsource — COMMENTED** [review]
Nice! I found a working link, but it created more questions; I let you give me your opinion.

File: rules/S1075/resources.adoc (hunk 1 above)
Comment (leonardo-pilastri-sonarsource) [reply rules/S1075/resources.adoc:3]: Indeed, now that I can see the content of the page it looks like it's not really related. Also in the Automated Detection table at the bottom of the page, the row for SonarQube does not mention this rule, but only S1313 and S2068.
So it makes sense to remove this resource and instead elaborate a bit on why URI can be considered sensitive info
👍

**Review by nicolas-gauthier-sonarsource — APPROVED** [review]
LGTM! The CI error on links validation doesn't seem related to this PR

**Pull request merged**